src/rpc/util.h

```diff
@@ -38,6 +38,11 @@ enum class TransactionError;
 struct FlatSigningProvider;
 struct bilingual_str;
 
+namespace node
+{
+class Warnings;
+} // namespace node
```

Comment:
I don't think this is needed.

Reply:
I meant the forward-declaration itself. Is it used for something?

Reply:
Oh damn it, I misread your comment. No, it isn't. This is a leftover from an earlier approach. Removed now, thanks.